ui/src/user/Login.tsx

```diff
@@ -61,6 +77,24 @@ class Login extends Component<Stores<'currentUser'>> {
         this.props.currentUser.login(this.username, this.password);
     };
 
+    private register = async (name: string, pass: string) => {
+        await axios.post('/registration', {name, pass});
```

Comment:
```suggestion
        await axios.post(config.get('url') + 'registration', {name, pass});
```
With just / requests will fail / be missrouted if gotify is [proxied to a subpath](https://gotify.net/docs/nginx#at-a-subpath).